docs/general/manage-tokens/staking-and-delegating.md

```diff
@@ -1,63 +1,92 @@
 # Staking and Delegating
 
-Staking and Delegation on the Oasis Network is a wonderful way to hold your ROSE
-tokens.
-Here are a few key resources to get started!
+The Oasis network is a proof-of-stake network. This means that the **voting power
+in the network is determined by the amount of tokens an entity posses**. For
+example, the amount of tokens determines, how frequent the validator will be
+elected to propose a new block. And each epoch, the staking reward is
+distributed among the validators based on the amount of the tokens they
+*stake*. You can check out the **current staking rewards** in the [Token metrics
+chapter][current staking rewards].
 
-## Rewards and Tokenomics
+But it's not just the validators that can stake. You can *delegate* your tokens
+to a validator and earn **passive income**, when the validator receives the
+staking reward. Of course, the validator may cut his share (the *commission
+fee*) for running the validator node hardware, but in essence staking **improves
+the security of the network** since by paying the commission fee, you reward
+good validators and expel the malicious ones.
 
```

Comment:
Should also say that if a validator misbehaves they can be slashed so you can actually lose a portion of your tokens if you delegate to a bad validator.